marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {CSSProperties} from "react";
+import {AutoComplete, Dropdown, Icon, Menu} from "antd";
+import styles from "./graph-view.module.scss";
+import {ModelingTooltips} from "../../../config/tooltips.config";
+import {MLTooltip, MLInput, MLButton} from "@marklogic/design-system";
+import {DownOutlined} from "@ant-design/icons";
+import PublishToDatabaseIcon from "../../../assets/publish-to-database-icon";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faFileExport} from "@fortawesome/free-solid-svg-icons";
+
+type Props = {
+  entityTypes: any;
+};
+
+const GraphView: React.FC<Props> = (props) => {
+
+  const publishIconStyle: CSSProperties = {
+    width: "20px",
+    height: "20px",
+    fill: "currentColor",
+  };
+
+  const filter = <AutoComplete
+    className={styles.filterInput}
+    dataSource={[]}
+    aria-label="graph-view-filter-autoComplete"
+    placeholder={"Filter"}
+  >
+    <MLInput aria-label="graph-view-filter-input" suffix={<Icon className={styles.searchIcon} type="search" theme="outlined" />} size="small"></MLInput>
+  </AutoComplete>;
+
+  const menu = (
+    <Menu>
+      <Menu.Item key="addNewEntityType">
+        <span aria-label={"addNewEntityTypeOption"}>Add new entity type</span>
+      </Menu.Item>
+      <Menu.Item key="addNewRelationship">
+        <span aria-label={"addNewRelationshipOption"}>Add new relationship</span>
+      </Menu.Item>
+    </Menu>
+  );
+
+  const headerButtons = <span className={styles.buttons}>
+    <span>
+      <Dropdown
+        overlay={menu}
+        trigger={["click"]}
+        overlayClassName={styles.stepMenu}
+        placement="bottomRight"
+      >
+        <div className={styles.addButtonContainer}>
+          <MLButton aria-label="add-entity-type-relationship" size="default" type="primary">
+            Add{" "}
```

Comment:
{" "} seems like a sketchy way to add space, it's not easily adjustable. Maybe wrap the "Add" in a tag and specify the right spacing via CSS? While we're at it, can we also clean up some of the indenting here (closing tags aligned)?